src/test/java/MapersTests/MapCharacterTests.java

```diff
@@ -0,0 +1,36 @@
+    private final MapCharacterClass testObject = new MapCharacterClass();
+    private final String badOutputMapper = "{\"ability_score\":{\"index\":\"dex\",\"name\":\"DEX\",\"url\":\"/api/ability-scores/dex\"},\"noslda\":2}";
+
+    @Test
+    public void mapperHappyPath() {
+        String resultForString = testObject.mapCharacterClass(outputResponseBody).toString();
+        Assertions.assertEquals(outputMapper, resultForString);
+    }
+
+    @Test
+    public void mapperUNHappyPath() {
+        Assertions.assertNotEquals(outputMapper, " ");
+    }
+
+    @Test
+    public void mapperUNHappyPathNull() {
+        String resultForString = testObject.mapCharacterClass(outputResponseBody).toString();
+        Assertions.assertNotEquals(null, resultForString);
```

Comment:
Checking for null is not a good check

Reply:
Better is empty string?

Reply:
Anything could be null, or an empty String. This does not test anything specific, same as other places

Reply:
ok, corrected!